### ISSUE TITLE
FLIList error

### ISSUE THREAD

**Issue by thiosady** [cversek/python-FLI#7]
I have this error with FLIList while testing lib.py:

```
>>>domain = flidomain_t(FLIDOMAIN_USB)
>>>names = POINTER(char_p)()
>>>libfli.FLIList(domain, byref(names))

                File "C:/Users/.../pythonFLI/lib.py", line 335, in chk_err
                                raise FLIError(msg)

              FLIError: Invalid argument
```

Therefore, I tried:
```
>>> names = POINTER(POINTER(char_p))()
>>> libfli.FLIList(domain, names)
                WindowsError: exception: access violation writing 0x0000000000000000

```
Any hints?

**Comment by cversek** [comment]
Take a look at the [find_devices](https://github.com/cversek/python-FLI/blob/master/src/FLI/device.py#L47) method of the device module for an example of how to call 'FLIList'.

**Comment by thiosady** [comment]
It raise the same error when I run device.py:
```
devs = USBDevice.find_devices()
  File "C:/Users/.../device.py", line 51, in find_devices
    cls._libfli.FLIList(cls._domain, byref(tmplist))      #allocates memory 
  File "lib.py", line 335, in chk_err
    raise FLIError(msg)
lib.FLIError: Invalid argument
```

**Comment by cversek** [comment]
It's quite possible that FLI has changed their API since I wrote that code, so I would check their documentation.  I am not associated with FLI, nor do I currently have access to any of their devices.  If everything seems to be the same, I would suggest that there might be an issue with the behavior of `ctypes` on Windows.  I developed the code mainly to work on Linux (~3 years ago) and the project was sidelined by the US government shutdown.
